src/aasthaJJun20/ArraysUnionIntersection.java

```diff
@@ -0,0 +1,80 @@
+package aasthaJJun20;
+
+public class ArraysUnionIntersection {
+	// This method is written just to display the array elements on console
+	private void displayArray(double[] input) {
+		System.out.print("{ ");
+		for (int index = 0; index < input.length; index++) {
+			System.out.print(input[index] + "   ");
+		}
+		System.out.println("}");
+	}
+
+	// Below method is written to search a particular element in the provided(input)
+	// array
+	private boolean searchInArray(double[] input, double element) {
+		for (int index = 0; index < input.length; index++) {
+			if (input[index] == element)
+				return true;
+		}
+		return false;
+	}
+
+	// Program 1: Find union of two arrays.
+	public double[] getUnionOfArray(double[] arr1, double[] arr2) {
+		double[] unionArray = new double[arr1.length + arr2.length];
+		int size = 0;
+		for (int index = 0; index < arr1.length; index++) {
+			if (searchInArray(unionArray, arr1[index]) == false) {
```

Comment:
searchArray length is sum of two array length, hence for loop should be iterated over searchArray length time to search every element. we can use little trick here to reduce number of comparison.

// Below method is written to search a particular element in the provided(input)
	// array
	private boolean searchInArray(double[] input, double element, int size) {
		for (int index = 0; index < size; index++) {
			if (input[index] == element)
				return true;
		}
		return false;
	}

Reply:
Yes, Krishna I was thinking of it, that how can I reduce the number of iterations, this is the perfect solution to it.
Thanks a lot. Implementing it now :)